src/standalone/Vegetation/PlantHydraulics.jl

```diff
@@ -269,8 +290,46 @@ ClimaLSM.auxiliary_types(model::PlantHydraulicsModel{FT}) where {FT} = (
     NTuple{model.n_stem + model.n_leaf, FT},
     NTuple{model.n_stem + model.n_leaf, FT},
     FT,
+    NamedTuple{(:root, :stem, :leaf), Tuple{FT, FT, FT}},
 )
 
+
+"""
+    set_canopy_parameter_field!(component::PlantHydraulics{FT},
```

Comment:
could make more modular once we have more parameters needing updating.....

Reply:
I agree, it would be nice for this to be less hardcoded. But we can address maybe adding an abstract type for prescribed parameters once we add more and need to generalize

Reply:
just a reminder that `parameter` naming feels a bit weird as area index will eventually be a prognostic variable (from leaf carbon pool, resulting from leaf growth (NPP*allocation to leaf) and accounting for leaf mass per area). 

that being said, the name is ok with me for now! just to keep in mind and maybe change later (or not).

Reply:
I left a note in the docs of CanopyModel

Note that the canopy height is specified as part of the 
PlantHydraulicsModel, along with the area indices of the leaves, roots, and
stems. Eventually, when plant biomass becomes a prognostic variable (by
integrating with a carbon model), some parameters that are specified here will be
treated differently.

Reply:
I also changed this to be `update_canopy_prescribed_field` and `set_canopy_prescribed_field`